doc/user/content/get-started/quickstart.md

```diff
@@ -43,323 +59,740 @@ Materialize.
+      | ----- | ------ | ------------------ | -------------------------- | ------ | ----- | ---------- | ------ | -------------------------- |
+      | 15575 | 158    | Signed Memorabilia | 2024-07-25 20:30:25.085+00 | 155751 | 215   | 15575      | 27     | 2024-07-25 20:30:16.085+00 |
+      | 15575 | 158    | Signed Memorabilia | 2024-07-25 20:30:25.085+00 | 155750 | 871   | 15575      | 63     | 2024-07-25 20:30:15.085+00 |
+      | 15575 | 158    | Signed Memorabilia | 2024-07-25 20:30:25.085+00 | 155752 | 2608  | 15575      | 16     | 2024-07-25 20:30:17.085+00 |
+      ```
+
+      As new data enters the system, above `SELECT` queries must be rerun to get
+      the recent results.
+
+      By creating a view to save repeated queries and then <redb>indexing</redb>
+      the view, Materialize can <redb>compute and then, as new data is ingested,
+      incrementally update</redb> view results in memory.
+
+      Using a query to find winning bids for auctions, subsequent steps in this
+      quickstart show how Materialize uses views and indexes to provide
+      immediately available up-to-date results for various queries.
 
-## Step 2. Use indexes for speed
+## Step 3. Create a view to find winning bids.
 
-**Operational work requires interactive access to data as soon as it's
-  available**. To identify potential auction flippers, you need to keep track
-  of the winning bids for each completed auction.
+A [view](/concepts/views/) is a saved name for the underlying `SELECT`
+statement, providing an alias/shorthand when referencing the query. The
+underlying query is not executed during the view creation; instead, the
+underlying query is executed when the view is referenced.
 
-1. Create a [**view**](/concepts/views/#views) that
-joins data from `auctions` and `bids` to get the bid with the highest `amount`
-for each auction at its `end_time`.
+Assume you want to find the winning bids for auctions that have ended. The
+winning bid for an auction is the highest bid entered for an auction before the
+auction ended. As new auction and bid data appears, the query must be rerun to
+get up-to-date results.
+
+1. Using the [`CREATE VIEW`](/sql/create-view/) command, create a
+   [**view**](/concepts/views/ "Saved name/alias for a query") `winning_bids`
+   for the query that finds winning bids for auctions that have ended.
+
+   ```mzsql
+   CREATE VIEW winning_bids AS
+   SELECT DISTINCT ON (a.id) b.*, a.item, a.seller
+   FROM auctions AS a
+    JOIN bids AS b
+      ON a.id = b.auction_id
+   WHERE b.bid_time < a.end_time
+     AND mz_now() >= a.end_time
+   ORDER BY a.id,
+     b.amount DESC,
+     b.bid_time,
+     b.buyer;
+   ```
+
+   [`SELECT DISTINCT ON
+   (...)`](https://www.postgresql.org/docs/current/sql-select.html#SQL-DISTINCT)
+   is a PostgreSQL expression that keeps only the first row of each set of
+   matching rows.
```

Comment:
Will link to idiomatic materialize sql once https://github.com/MaterializeInc/materialize/pull/29643 is merged.